### PR TITLE
grpc,google-protobuf: migrate to homebrew-core

### PR DESCRIPTION
**Pull request by chenrui333**
relates to:
- https://github.com/Homebrew/homebrew-core/pull/9942
- #97

### CODE REVIEW

**Comment by chenrui333** [comment]
cc @jtattermusch @mehrdada @sdamodharan

**Review by jtattermusch — COMMENTED** [review]
Can you please provide more context?

It seems that since the formulas in this repo haven't been updated for the past 5years or so, the grpc/homebrew-grpc repository should basically be considered deprecated.

Perhaps we should delete the formulas from this repository and have the README.md basically say "don't use this, use the community-maintained homebrew formulas (that seems to be significantly more up-to-date)".
Is that aligned with that this PR is trying to do?

File: tap_migrations.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "grpc": "homebrew/core",
```
Comment (jtattermusch) [inline tap_migrations.json:2]: is this the same formula as  https://formulae.brew.sh/formula/grpc

File: tap_migrations.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+  "google-protobuf": "homebrew/core/protobuf"
```
Comment (jtattermusch) [inline tap_migrations.json:3]: is this the same formula as https://formulae.brew.sh/formula/protobuf#default?

**Comment by jtattermusch** [comment]
I created https://github.com/grpc/homebrew-grpc/pull/112 to update the readme.

**Comment by chenrui333** [comment]
@jtattermusch yeah, the context is basically to help migrate the formulae users to use the well maintained ones in homebrew-core, which leverages something called `tap_migrations.json`.

In other words, if the users installed the formulae using the taps, when they do `brew upgrade xxx`, they will automatically get redirected to use the formulae in homebrew-core.

Let me know if it makes better sense now.

**Review by jtattermusch — APPROVED** [review]

**Comment by jtattermusch** [comment]
LGTM since this migrates the users of the very old and deprecated formulas to the community-provided ones (which are part of homebrew-core):
https://formulae.brew.sh/formula/grpc
https://formulae.brew.sh/formula/protobuf

**Pull request merged**